src/java.base/share/classes/java/util/UUID.java

```diff
@@ -463,7 +468,54 @@ public long node() {
      */
     @Override
     public String toString() {
-        return jla.fastUUID(leastSigBits, mostSigBits);
```

Comment:
`JavaLangAccess.fastUUID` method now can be removed

Reply:
it fixed.